src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -126,7 +127,7 @@ def check_align_starting_time_internal(self):
         expected_timestamps = unaligned_timestamps + starting_time
         assert_array_equal(x=aligned_timestamps, y=expected_timestamps)
 
-    def check_align_starting_time_external(self):
+    def check_align_timestamps_internal(self):
```

Comment:
Can you add a short string to these methods? I can't understand what they are doing.

Reply:
Looking back over the mixin methods, there's a lot that could be improved...

Would you be open to working together on a follow-up that cleans up and enhances those core testing methods?

Note that in this PR, we really don't use the mixins in the core testing suite and instead it's all in the `test_behavior/test_audio_interface` which uses generated files (since that's where the majority of tests were prior to this PR anyway)

Reply:
Yes, let's, I will schedule something with you.